tests/compiler/tdebugutils.nim

```diff
@@ -1,6 +1,6 @@
 discard """
   joinable: false
-  disabled: true
+    disabled: true
```

Comment:
was born disabled. I guess it is used only for demonstration purposes? 
		from my current understanding, these type of tests should be coalesced into the specs?

Reply:
i believe @quantimnot is working on [someting among those lines](https://github.com/saem/vscode-nim/issues/60#issuecomment-958635999), but otherwise, i surely can.

Reply:
different debugging, this is glorified echo/print based debugging.

Reply:
I'll be keeping this as disabled for now. I will probably get around to this documentation soon&trade;